postman/scripts/sendMessageOnL2.ts

```diff
@@ -2,9 +2,9 @@ import { ContractTransactionReceipt, Overrides, JsonRpcProvider, Wallet } from "
 import { config } from "dotenv";
```

Comment:
@VGau Same, should we move this script to sdk? I dun see how it's relevant to postman except as a handy tool for testing local postman

Reply:
This script is specific to the Postman so I think we should put it inside the SDK ? WDYT ?